waitress/settings/production.py

```diff
@@ -1,16 +1,19 @@
 # -*- coding: utf-8 -*-
-from .base import *
 import dj_database_url
 
+from .base import *
+
+
 PRODUCTION_DOMAIN = "waitressandela.herokuapp.com"
-STAGING_DOMAIN = "waitress-staging.herokuapp.com"
 
-DEBUG = True
+DEBUG = False
```

Comment:
Shouldn't this be set based on env variable condition like `env != 'production'?`